posts/near-edu-fellowship-recap.md

```diff
@@ -0,0 +1,53 @@
+# Our time as NEAR Education Fellows: Reflections, Retrospective, Accomplishments, What's Next
+
+Aha Labs joined the NEAR Education Fellowship program in January 2022. This program has changed structure; our time with it ended in May. Some highlights:
```

Comment:
```suggestion
Aha Labs joined the NEAR Education Fellowship program in January 2022. This program has since changed structure; our time with it ended in May. Some highlights:
```

Reply:
seems optional; I'd rather go with fewer words where possible